py/moq-rs/moq/publish.py

```
@@ -440,6 +486,19 @@ def publish_audio(
         """Publish a raw-audio track with an in-process Opus encoder."""
         return AudioProducer(self._inner.publish_audio(name, input, output))
 
+    def publish_video(
+        self,
+        input: VideoEncoderInput,
```

Ruff (0.16.0)

[error] Function argument `input` is shadowing a Python builtin

(A002)